### DISCUSSION TITLE
[keyball44]ランドの焦げと、剥がれをそれぞれ被覆銅線でつなげたい

### DISCUSSION THREAD

**Discussion by shima00yasu** [Yowkees/keyball#207] [Q&A]
ハンダ初心者です。

1枚めはランドが焦げてしまい、2枚めはランドが剥がれてしまいました。
それぞれ被覆導線でつなぎたいですが、どことつなげればいいか教えていただきたいです。

お忙しいところすみませんが、よろしくおねがいいたします。
![IMG_3292](https://user-images.githubusercontent.com/99468552/229656994-bf355a43-228b-49df-ada1-40f455616ab2.png)
![IMG_3291](https://user-images.githubusercontent.com/99468552/229657009-77fc69c5-0bd8-451b-ab47-a66c245b42f6.png)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。状況、了解しました。
![44socket_fix](https://user-images.githubusercontent.com/84209772/229945561-4bb996a4-01d8-4035-98b5-5dffafd6969d.jpg)
![44diode_fix](https://user-images.githubusercontent.com/84209772/229945566-638a9ca4-b9d4-467d-93b3-05580685237a.jpg)
画像の同色どうしを導線で接続していただければ、修理可能です。
なお、キーソケットは固定力が弱いと思いますので、両面テープ等で張り付けた方が良いかと思います。

Keyball44の回路図、基板図は下記にて公開しておりますので、こちらで線を辿っていただくと修理内容が理解しやすいかと思います。
https://github.com/Yowkees/keyball/tree/main/keyball44/design_data
お手数ですがご対応お願いいたします。

**Reply by shima00yasu** [reply]
ありがとうございます！
回路図、基盤図も見てみます。
